line-plot-data.php

```diff
@@ -0,0 +1,45 @@
+<?php
+require_once('mysql_login.php');
+
+$conn = new PDO("mysql:host=$servername;dbname=$dbname;charset=utf8mb4", $username, $password);
+
+$requestData= $_REQUEST;
+
+$startDate = date('Y-m-d');
+$endDate = date('Y-m-d');
```

Comment:
We need to make sure that the timelimit of what's being requested isn't too big, let's start with a limit of two weeks, otherwise the data will be massive